### PR TITLE
Dịch "dropout_vn" - Phần 6

### LINKED ISSUE #948: Dịch "dropout_vn" - Phần 6
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/dropout_vn.md#L409-L447

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by nguyenduydu**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #948

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 1]
```diff
@@ -420,7 +420,9 @@ d2l.train_ch3(net, train_iter, test_iter, loss, num_epochs, trainer)
 * Dropout is only used during training.
 -->
 
-*dịch đoạn phía trên*
+* Ngoài phương pháp kiểm soát số chiều và kích thước của vector trọng số thì dropout cũng là một công cụ nữa để tránh tình trạng quá khớp. Thông thường thì cả ba cách được sử dụng cùng nhau.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
* Ngoài phương pháp kiểm soát số chiều và kích thước của vector trọng số, dropout cũng là một công cụ khác để tránh tình trạng quá khớp. Thông thường thì cả ba cách được sử dụng cùng nhau.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 2]
```diff
@@ -420,7 +420,9 @@ d2l.train_ch3(net, train_iter, test_iter, loss, num_epochs, trainer)
...
+* Dropout thay thế giá trị kích hoạt $h$ bằng một biến ngẫu nhiên $h'$ với giá trị kỳ vọng $h$ và với phương sai được tính toán bởi xác suất dropout $p$.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
* Dropout thay thế giá trị kích hoạt $h$ bằng một biến ngẫu nhiên $h'$ với giá trị kỳ vọng $h$ và phương sai bằng xác suất dropout $p$.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 3]
```diff
@@ -442,7 +444,15 @@ What happens when dropout and weight decay are used at the same time? Are the re
 Can you develop a method that outperforms dropout on the FashionMNIST dataset (for a fixed architecture)?
 -->
 
-*dịch đoạn phía trên*
+1. Điều gì xảy ra nếu bạn thay đổi xác suất dropout của tầng 1 và 2? Cụ thể, điều gì xảy ra nếu bạn tráo đổi xác suất của hai tầng này? Thiết kế một thí nghiệm để trả lời những câu hỏi này, mô tả các kết quả một cách định lượng và tóm tắt các bài học định tính.
+2. Tăng số lượng epoch và so sánh các kết quả thu được khi sử dụng và khi không sử dụng dropout.
+3. Tính toán phương sai của các giá trị kích hoạt ở mỗi tầng ẩn khi sử dụng và khi không sử dụng dropout. Vẽ một biểu đồ để cho thấy giá trị phương sai này cả hai mô hình phát triển như thế nào theo thời gian.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
3. Tính toán phương sai của các giá trị kích hoạt ở mỗi tầng ẩn khi sử dụng và khi không sử dụng dropout. Vẽ biểu đồ thể hiện cách giá trị phương sai này thay đổi theo thời gian cho cả hai mô hình.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 4]
```diff
@@ -442,7 +444,15 @@ What happens when dropout and weight decay are used at the same time? Are the re
...
+4. Tại sao dropout thường không được sử dụng tại bước kiểm tra?
+5. Sử dụng mô hình trong phần này làm ví dụ, so sánh hiệu quả của việc sử dụng dropout và phân rã trọng số.
+Điều gì xảy ra khi dropout và phân rã trọng số được sử dụng cùng một lúc? Hai phương pháp này bổ trợ cho nhau, làm giảm hiệu quả của nhau hay (tệ hơn) loại trừ lẫn nhau?
+6. Điều gì xảy ra nếu chúng ta áp dụng dropout cho các trọng số riêng lẻ của ma trận trọng số thay vì các giá trị kích hoạt?
+7. Phát minh ra một kỹ thuật khác với kỹ thuật dropout tiêu chuẩn để thêm nhiễu ngẫu nhiên ở mỗi tầng.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
7. Hãy phát minh một kỹ thuật khác với kỹ thuật dropout tiêu chuẩn để thêm nhiễu ngẫu nhiên ở mỗi tầng.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 5]
```diff
@@ -442,7 +444,15 @@ What happens when dropout and weight decay are used at the same time? Are the re
...
+Bạn có thể phát triển một phương pháp cho kết quả vượt trội hơn dropout trên bộ dữ liệu FashionMNIST không (với một kiến trúc cố định)?
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]: từ `vượt trội` cảm giác hơi đỉnh cao quá :)))
```suggestion
Bạn có thể phát triển một phương pháp cho kết quả tốt hơn dropout trên bộ dữ liệu FashionMNIST không (với cùng một kiến trúc)?
```

**Comment by nguyenduydu** [comment]
@minhduc0711  Cám ơn bạn đã góp ý nhé.

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 6]
```diff
@@ -486,4 +496,4 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 6 -->
-*
+* Nguyễn Duy Du
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
* Nguyễn Duy Du
* Phạm Minh Đức
```
